text/pvm.tex

```diff
@@ -570,29 +570,31 @@ \subsubsection{Instructions with Arguments of Three Registers}
   \midrule
   \endhead
   170&\token{add\_32}&0&$\reg'_D = \sext_4((\reg_A + \reg_B) \bmod 2^{32})$\\ \mrule
-  171&\token{sub\_32}&0&$\reg'_D = \sext_4((\reg_A + 2^{32} - \reg_B) \bmod 2^{32})$\\ \mrule
+  171&\token{sub\_32}&0&$\reg'_D = \sext_4((\reg_A + 2^{32} - (\reg_B \bmod 2^{32})) \bmod 2^{32})$\\ \mrule
   172&\token{mul\_32}&0&$\reg'_D = \sext_4((\reg_A \cdot \reg_B) \bmod 2^{32})$\\ \mrule
   173&\token{div\_u\_32}&0&$\reg'_D = \begin{cases}
     2^{64} - 1 &\when \reg_B = 0\\
-    \sext_4(\floor{\reg_A \div \reg_B}) &\otherwise
+    \floor{(\reg_A \bmod 2^{32}) \div (\reg_B \bmod 2^{32})} &\otherwise
```

Comment:
The `\reg_B = 0` here still needs a modulo, otherwise we'll get division by zero if e.g. `ω_B = 0x00000001_00000000`